core/src/retransmit_stage.rs

```diff
@@ -119,6 +138,13 @@ fn update_retransmit_stats(
                 i64
             ),
         );
+        let mut packets_by_slot = stats.packets_by_slot.lock().unwrap();
+        info!("retransmit: packets_by_slot: {:?}", packets_by_slot);
```

Comment:
output might be a little easier to read with maps if we pretty print with `{:#?}` over `{:?}`